lib/Doctrine/DBAL/Types/ConversionException.php

```diff
@@ -99,4 +99,16 @@ static public function conversionFailedInvalidType($value, $toType, array $possi
             implode(', ', $possibleTypes)
         ));
     }
+
+    static public function conversionFailedSerialization($value, $format, $error, \Exception $previous = null)
```

Comment:
@Taluu yeah sorry then for the extra effort. Could you be so kind to just use `conversionFailedFormat()` then? Also you might just put the error message into `$expectedFormat`. Like:

```php
<?php

// snippet

if (JSON_ERROR_NONE !== json_last_error()) {
    throw ConversionException::conversionFailedFormat(
        $value,
        Type::JSON,
        'json (' . $this->getLastErrorMessage() . ')'
    );
}
```